src/main/java/net/fabricmc/loader/api/MappingResolver.java

```diff
@@ -21,63 +21,67 @@
 /**
  * Helper class for performing mapping resolution.
  *
- * Note: The target namespace (the one being mapped to) for mapping (or the
+ * <p><strong>Note</strong>: The target namespace (the one being mapped to) for mapping (or the
  * source one for unmapping) is always implied to be the one Loader is
- * currently operating in.
+ * currently operating in.</p>
+ *
+ * <p>All the {@code className} used in this resolver are in <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.1">binary names</a>,
+ * such as {@code "mypackage.MyClass$Inner"}.</p>
```

Comment:
are you sure? I have a feeling its /

Reply:
I think it is `/` as always used in mappings, but the old doc says it is dot format (see changeset below)

Reply:
https://github.com/FabricMC/fabric-loader/blob/9f6b85907d239a3e40e1476140a69998cf06f7ca/src/main/java/net/fabricmc/loader/FabricMappingResolver.java#L106-L108

Well it does take binary/dot format, rip (java's own `Class.forName` uses binary/dot format)